tests/cross_functional/system_test/test_nfs_export_in_out_cluster.py

```
@@ -0,0 +1,2001 @@
+        # Step 5: Mount restored PVC on NFS client and verify data integrity
+        log.info(
+            "Step 5: Mounting restored PVC on NFS client for data integrity verification"
+        )
+
+        # Get NFS share details for restored PVC
+        fetch_restored_vol_name_cmd = (
+            "get pvc " + restored_pvc_name + " --output jsonpath='{.spec.volumeName}'"
+        )
+        restored_vol_name = self.pvc_obj.exec_oc_cmd(fetch_restored_vol_name_cmd)
+        log.info(
+            f"For restored PVC {restored_pvc_name} volume name is, {restored_vol_name}"
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)